src/utils/filterHooks/index.ts

```diff
@@ -0,0 +1,121 @@
+import _ from 'lodash';
+import { isArray } from 'src/types/utils';
+
+type FilterValue = (string | number | null)[] | boolean | null;
+export type FiltersType = Record<string, FilterValue>;
+
+// Convert an array to a comma separated string and other values to a string
+// Array values are sorted, so we can detect equality easier elsewhere
+export const normalizeValue = (value: FilterValue): string => {
```

Comment:
just curious.. would a better pattern be
`?filter_facility_id=123&filter_facility_id=345` and use `query.getAll('filter_facility_id')`
over
`?filter_facility_ids=123,345` 
I understand it's a bigger code change but just for discussion.

Reply:
Yea I was thinking about that, wouldn't mind making the change. I think the way you propose is probably better in the long run because if the filter values end up having a comma things will have to get rewritten anyways.

Reply:
Got it, if the rewrite is bigger than anticipated.. we can certainly track it for future tech investment. The pros to keeping it as-is --> already validated and no comma in value use-case yet.